client/blocks/login/continue-as-user.jsx

```diff
@@ -98,14 +99,14 @@ export default function ContinueAsUser( {
 						</button>
 					</div>
 				</div>
-				<Button
+				<A8CButton
```

Comment:
I'm not familiar with these other product screens so for now I've left them and focused on dotcom